packages/vm/CHANGELOG.md

```diff
@@ -6,6 +6,54 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 5.9.0 - 2022-04-14
+
+### EIP-3651: Warm COINBASE (Shanghai CFI EIP)
+
+Small EIP - see [EIP-3651](https://eips.ethereum.org/EIPS/eip-3651) considered for inclusion (CFI) in Shanghai to address an initially overpriced `COINBASE` access, PR [#1814](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1814).
+
+EIP can be activated manually with:
+
+```typescript
+const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.London, eips: [ 3651 ] })
+```
+
+### EIP-1153: Transient Storage Opcodes
+
+Experimental implementation of [EIP-1153](https://eips.ethereum.org/EIPS/eip-1153), see PR [1768](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1768), thanks to [Mark Tyneway](https://github.com/tynes) from Optimism for the implementation! ❤️
+
+The EIP adds opcodes for manipulating state that behaves identically to storage but is discarded after every transaction. This makes communcation via storage (`SLOAD`/`SSTORE`) more efficient and would allow for significant gas cost reductions for various use cases.
+
+Hardfork inclusion of the EIP has been extensively discussed along [ACD 135, April 1 2022](https://github.com/ethereum/pm/issues/500).
+
+EIP can be activated manually with:
+
+```typescript
+const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.London, eips: [ 1153 ] })
+```
+
+### Custom Precompiles (L2 Support)
+
+It is now possible to add, override or delete precompiles on the VM with a new `customPrecompiles` option, see PR [#1813](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1813). This allow for further behavioral VM customization in addition to the lately added `customOpcodes` option, which can be useful for L2 solutions or EVM based side chains or L1.
```

Comment:
```suggestion
It is now possible to add, override or delete precompiles in the VM with a new `customPrecompiles` option, see PR [#1813](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1813). This allow for further customization of VM behavior in addition to the recently added `customOpcodes` option, which can be useful for L2 solutions or EVM based side chains or L1.
```

Should we also refer to the VM as "EVM" here since it's really updating the behavior of the EVM itself and not the outer VM, right?